src/interface/scatterplot.jl

```diff
@@ -28,6 +28,8 @@ Arguments
 - **`name`** : Annotation of the current drawing to be displayed
   on the right
 
+- **`markers`** : choose from (:circle, :rect, :diamond, :hexagon, :cross, :xcross, :utriangle, :dtriangle, :rtriangle, :ltriangle, :pentagon, :star4, :star5, :star6, :star8, :vline, :hline, :+, :x) or use a `Char` or unit length `String`
```

Comment:
```suggestion
**`marker`** : choose from $(keys(MARKERS)) or use a `Char` or unit length `String`
```

Reply:
Nice, I didn't know we could do that in a docstring.